Resources/Private/JavaScript/Components/Dialog/index.js

```diff
@@ -30,6 +31,9 @@ const Dialog = props => {
                             id="neos__modal__closeModal"
                             onClick={e => executeCallback({e, cb: onRequestClose})}
                             />
+                        <div className={style.dialog__title}>
```

Comment:
Yes, I will switch to it. I want to collect all typography styles from current neos interface, and assign them to h1-h6 Headline components, would that be a way to go?